apps/api/src/routes/account.routes.ts

```diff
@@ -1,454 +1,597 @@
+   *                               type: string
+   *                               description: Contact email for the agent
+   *                             twitter:
+   *                               type: string
+   *                               description: Twitter handle
+   *                     createdAt:
+   *                       type: string
+   *                       format: date-time
+   *                       description: Team creation timestamp
+   *                     updatedAt:
+   *                       type: string
+   *                       format: date-time
+   *                       description: Team update timestamp
+   *       401:
+   *         description: Unauthorized - Missing or invalid authentication
+   *       404:
+   *         description: Team not found
+   *       500:
+   *         description: Server error
+   */
+  router.put("/profile", controller.updateProfile);
 
-/**
- * @openapi
- * /api/account/balances:
- *   get:
- *     tags:
- *       - Account
- *     summary: Get token balances
- *     description: Get all token balances for the authenticated team
- *     security:
- *       - BearerAuth: []
- *     responses:
- *       200:
- *         description: Team token balances
- *         content:
- *           application/json:
- *             schema:
- *               type: object
- *               properties:
- *                 success:
- *                   type: boolean
- *                   description: Operation success status
- *                 teamId:
- *                   type: string
- *                   description: Team ID
- *                 balances:
- *                   type: array
- *                   items:
- *                     type: object
- *                     properties:
- *                       token:
- *                         type: string
- *                         description: Token address
- *                       amount:
- *                         type: number
- *                         description: Token balance amount
- *                       chain:
- *                         type: string
- *                         enum: [evm, svm]
- *                         description: Blockchain type of the token
- *                       specificChain:
- *                         type: string
- *                         nullable: true
- *                         description: Specific chain for EVM tokens
- *       401:
- *         description: Unauthorized - Missing or invalid authentication
- *       500:
- *         description: Server error
- */
-router.get("/balances", AccountController.getBalances);
+  /**
+   * @openapi
+   * /api/account/reset-api-key:
+   *   post:
+   *     tags:
+   *       - Account
+   *     summary: Reset team API key
+   *     description: Reset the API key for the authenticated team. This will invalidate the current API key and generate a new one.
+   *     security:
+   *       - BearerAuth: []
+   *     parameters:
```

Comment:
It's tricky to tell with the way this diff is presented, but you've added back an older version of the spec here.   This is probably an artifact of a merge conflict resolution.
We don't want to include the `BearerAuth` security strategy and explicitly add the `Authorization` header.  
I can create a separate PR to update the spec if you'd like?

Reply:
Can I just grab the latest spec from `main`?

Reply:
Ok yes, I messed up the rebase of this file and ended up with duplicate routes defined, each with different spec comments. I removed the duplicate and grabbed the spec comment from main.